common/changes/miwhea-fabric-core-6_2017-03-21-16-51.json

```diff
@@ -0,0 +1,20 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Update to Fabric Core 6.0.1",
+      "type": "major"
```

Comment:
If someone is using Fabric React to get a copy of Fabric Core (via references into its `node_modules`) then this will be a breaking change for them, as many of Core's mixins and variable names have changed.